apps/hash-api/src/graphql/resolvers/integrations/linear/linear-teams.ts

```
@@ -0,0 +1,45 @@
+import { getLinearUserSecretByLinearOrgId } from "../../../../graph/knowledge/system-types/linear-user-secret";
+import { listTeams } from "../../../../integrations/linear";
+import {
+  LinearTeam,
+  QueryLinearTeamsArgs,
+  ResolverFn,
+} from "../../../api-types.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+
+export const linearTeamsResolver: ResolverFn<
+  Promise<LinearTeam[]>,
+  {},
+  LoggedInGraphQLContext,
+  QueryLinearTeamsArgs
+> = async (_, params, { dataSources, user, vault }) => {
+  const linearSecretEntity = await getLinearUserSecretByLinearOrgId(
```

We should throw an error explicitly if this doesn't exist